bin/ffscreencast

```diff
@@ -153,52 +179,58 @@ write_config() {
 			$(which mkdir) -p "${dir}"
 		fi
 
-		echo "# ~/.config/ffscreencast/ffscreencastrc" > "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default video container extension" >> "${conf}"
-		echo "# Alternatively: 'mp4' or 'avi'" >> "${conf}"
-		echo "OUTPUT_EXT=\"mkv\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default audio output codec" >> "${conf}"
-		echo "# Alternatively: 'pcm_s16le'" >> "${conf}"
-		echo "OUTPUT_ACODEC=\"libfaac\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default video output codec" >> "${conf}"
-		echo "# Alternatively: 'libx265'" >> "${conf}"
-		echo "OUTPUT_VCODEC=\"libx264\"" >> "${conf}"
-		echo >> "${conf}"
-
-
-		echo "# Default Screen recording arguments" >> "${conf}"
-		echo "S_ARGS=\"\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default audio recording arguments" >> "${conf}"
-		echo "A_ARGS=\"-ac 2\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default camera recording arguments" >> "${conf}"
-		echo "C_ARGS=\"\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default misc output arguments" >> "${conf}"
-		echo "O_ARGS=\"-crf 0 -preset ultrafast\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# Default recording behavior" >> "${conf}"
-		echo "RECORD_S=\"yes\"" >> "${conf}"
-		echo "RECORD_A=\"no\"" >> "${conf}"
-		echo "RECORD_C=\"no\"" >> "${conf}"
-		echo >> "${conf}"
-
-		echo "# What listed device number has been chosen to record?" >> "${conf}"
-		echo "CHOSEN_S_NUM=\"\"" >> "${conf}"
-		echo "CHOSEN_A_NUM=\"\"" >> "${conf}"
-		echo "CHOSEN_C_NUM=\"\"" >> "${conf}"
-		echo >> "${conf}"
+		{
+			echo "# ~/.config/ffscreencast/ffscreencastrc"
+			echo 
+
+			echo "# Default Directory to save video files" 
+			echo "DIR=\"$HOME/Videos\"" 
+			echo 
+
+			echo "# Default video container extension" 
+			echo "# Alternatively: 'mp4' or 'avi'" 
+			echo "OUTPUT_EXT=\"mkv\"" 
+			echo 
+
+			echo "# Default audio output codec" 
+			echo "# Alternatively: 'pcm_s16le'" 
+			echo "OUTPUT_ACODEC=\"libfaac\"" 
+			echo 
+
+			echo "# Default video output codec" 
+			echo "# Alternatively: 'libx265'" 
+			echo "OUTPUT_VCODEC=\"libx264\"" 
+			echo 
+
+
+			echo "# Default Screen recording arguments" 
+			echo "S_ARGS=\"\"" 
+			echo 
+
+			echo "# Default audio recording arguments" 
+			echo "A_ARGS=\"-ac 2\"" 
+			echo 
+
+			echo "# Default camera recording arguments" 
+			echo "C_ARGS=\"\"" 
+			echo 
+
+			echo "# Default misc output arguments" 
+			echo "O_ARGS=\"-crf 0 -preset ultrafast\"" 
+			echo 
+
+			echo "# Default recording behavior"
+			echo "RECORD_S=\"yes\""
+			echo "RECORD_A=\"no\""
+			echo "RECORD_C=\"no\""
+			echo 
+
+			echo "# What listed device number has been chosen to record?"
+			echo "CHOSEN_S_NUM=\"\""
+			echo "CHOSEN_A_NUM=\"\""
+			echo "CHOSEN_C_NUM=\"\""
+			echo 
+		} >> "${conf}"
```

Comment:
@Dark-Kernel FYI there's a subtle change in behavior here. Previously passing through this code would do

    echo "foo" > $conf # clobber

and then a series of

    echo "bar" >> $conf # append new file

Changing this to

    {
        echo "foo"
        echo "bar"
    } >> $conf

means we no longer clobber the old config, instead we'll append a new config to it.

I think you can just use

    {
        echo "foo"
        echo "bar"
    } > $conf # note, just a single `>`

to achieve the prior behavior.

Reply:
Yes, it's small mistake, i didn't noticed that one >  is extra, sorry 😅.

Reply:
@Dark-Kernel cool, it seems like an easy fix - but this PR doesn't contain it (yet - maybe you haven't had a chance to make it yet: totally understandable!)

Reply:
I've opened https://github.com/Dark-Kernel/ffscreencast/pull/1 which targets your branch - if merged it will resolve this `>>` -> `>` issue.